docs/projects/ad7606x_fmcz/index.rst

```diff
@@ -0,0 +1,450 @@
+JP9,JP11           A                Bypasses the amplifier mezzanine card
+JP12               B                Connects the V2- line to the J6 SMB connector
+JP13               B                Connects the V3- line to the J7 SMB connector
+JP14               B                Connects the V4- line to the J8 SMB connector
+P13                Inserted         Connects the V5- line to ground
+P14                Inserted         Connects the V8- line to ground
+P15                Inserted         Connects the V6- line to ground
+P16                Inserted         Connects the V7- line to ground
+S1                 On               Controls the OS0, OS1, and OS2 pins
+================== ================ ====================================
+
+IP list
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :git-hdl:`AD_EDGE_DETECT <main:library/common/ad_edge_detect.v>` **
+-  :git-hdl:`AXI_AD7606 <main:library/axi_ad7606x>` *
+-  :git-hdl:`AXI_CLKGEN <main:library/axi_clkgen>` **
+-  :git-hdl:`AXI_DMAC <main:library/axi_dmac>`
+-  :git-hdl:`AXI_HDMI_TX <main:library/axi_hdmi_tx>`
+-  :git-hdl:`AXI_I2S_ADI <main:library/axi_i2s_adi>`
+-  :git-hdl:`AXI_PWM_GEN <main:library/axi_pwm_gen>`
+-  :git-hdl:`AXI_SPDIF_TX <main:library/axi_spdif_tx>`
+-  :git-hdl:`AXI_SPI_ENGINE <main:library/spi_engine/axi_spi_engine>` **
+-  :git-hdl:`AXI_SYSID <main:library/axi_sysid>`
+-  :git-hdl:`SPI_ENGINE_EXECUTION <main:library/spi_engine/spi_engine_execution>` **
+-  :git-hdl:`SPI_ENGINE_INTERCONNECT <main:library/spi_engine/spi_engine_interconnect>` **
+-  :git-hdl:`SPI_ENGINE_OFFLOAD <main:library/spi_engine/spi_engine_offload>` **
+-  :git-hdl:`SYNC_BITS <main:library/util_cdc/sync_bits.v>` **
+-  :git-hdl:`SYSID_ROM <main:library/sysid_rom>`
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``*`` instantiated only for INTF=0 (parallel interface)
+   -   ``**`` instantiated only for INTF=1 (serial interface)
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+========================  ==========
+Instance                  Address
+========================  ==========
+axi_ad7606x_dma           0x44a30000
+spi_clkgen                0x44a70000
+ad7606_pwm_gen **         0x44b00000
+ad7606_pwm_gen *          0x44A60000
+spi_ad7616_axi_regmap **  0x44a00000
+axi_ad7606x *             0x44A00000
+========================  ==========
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``*`` instantiated only for SER_PAR_N=0 (parallel interface)
+   -   ``**`` instantiated only for SER_PAR_N=1 (serial interface)
+
+I2C connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 20 20 20 20 20
+   :header-rows: 1
+
+   * - I2C type
+     - I2C manager instance
+     - Alias
+     - Address
+     - I2C subordinate
+   * - PL
+     - iic_fmc
+     - axi_iic_fmc
+     - 0x4162_0000
+     - \-
```

Comment:
Replace \ - with  `---` (3 dashes), at 218 and 223

Reply:
Done.